cmd/av/pr_queue.go

```diff
@@ -0,0 +1,110 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"os"
+
+	"github.com/aviator-co/av/internal/actions"
+	"github.com/aviator-co/av/internal/avgql"
+	"github.com/shurcooL/graphql"
+	"github.com/spf13/cobra"
+)
+
+var prQueueFlags struct {
+	SkipLine 	bool
+	Targets		string
+}
+
+var prQueueCmd = &cobra.Command{
+	Use:          "queue",
+	Short:        "queue a pr for the current branch",
+	SilenceUsage: true,
+	Args:         cobra.NoArgs,
+	// error or reterr from emperror.dev/errors here?
+	RunE: func(cmd *cobra.Command, args []string) error {
+		variables, err := getMutationVariables()
+		if err != nil {
+			return err
+		}
+
+
+		// I have a feeling this would be better written inside of av/internals
+		client := avgql.NewClient()
+
+		var mutation struct {
+			QueuePullRequest struct {
+				UpdateMergeQueueStatusPayload struct {
+					PullRequest	graphql.String
+				}
+					
+			} `graphql:"githubRepository(input: {owner: $repoOwner, name:$repoName})"`
+		}
+		err = client.Mutate(context.Background(), &mutation, variables)
+		if err != nil {
+			return err
+		}
+		// We print to stderr everywhere, why?
+		// Also, should we surface anything else to the user? bar improving what we actually 
+		pr := mutation.QueuePullRequest.UpdateMergeQueueStatusPayload.PullRequest
+
+		fmt.Fprintf(
+			os.Stdout, 
+			string(pr),
+			"\n",
+		)
+		return nil
+	},
+}
+
+func getMutationVariables() (map[string]interface{}, error) {
```

Comment:
this function is in the `main` namespace so pls name it more descriptively. I'd honestly prefer just inlining this into `prQueueCmd.RunE` but I'm weird that way.

Reply:
Disagree on the in-lining, I think pulling this out improves readability.

This pattern is borrowed from `pr_status.go`, open to function renaming recommendations?

Reply:
What do you think about moving the actual mutation into this function then, and just returning the pull request info? Right now, this function is separate from the `var mutation` definition, but also it's very coupled to it (the keys in the map here have to match the variables used in the mutation). It's always a yellow/red-ish flag when you have that tight of coupling.

That being said, it's still within the same file, just ~20 lines away, so it's not the end of the world. If you just want to rename this function, we could just do `getQueuePullRequestMutationVariables`. It's obviously long but it's only used once so 🤷.

Reply:
I in-lined it.

Reply:
done